suite-common/wallet-core/src/accounts/accountsActions.ts

```diff
@@ -121,6 +121,19 @@ const updateAccount = createAction(
     },
 );
 
+const renameAccount = createAction(
+    `${actionPrefix}/renameAccount`,
+    (account: Account, accountLabel: string): { payload: Account } => ({
```

Comment:
Just pass `accountKey` and `accountLabel` and do all the work in reducer. Now you basically duplicated `updateAccount` action.

Reply:
fixed in e44e7fee9